lib/components/Blocks/Header/index.tsx

```diff
@@ -0,0 +1,31 @@
+import { FlexBox } from "@/components/Layout/_FlexBox"
+import { Avatar, AvatarFallback, AvatarImage } from "@/ui/avatar"
+import { forwardRef } from "react"
+
+interface HeaderProps {
+  title: string
+  subtitle: string
+  src: string
+  alt: string
+}
+
+export const Header = forwardRef<HTMLDivElement, HeaderProps>(
+  ({ title, subtitle, src, alt }, ref) => {
+    return (
+      <FlexBox ref={ref} className="px-10 py-6">
+        <Avatar size="xlarge">
+          <AvatarImage src={src} alt={alt} />
+          <AvatarFallback>{alt}</AvatarFallback>
+        </Avatar>
+        <FlexBox className="flex-col gap-4 pl-5">
+          <h1 className="pt-2 text-[1.69rem]/[1.6rem] font-medium text-foreground">
```

Comment:
This font size is a bit weird, I would try to use one of the existing sizes even if it looks a bit off, and then we normalize everything at once. I think Tailwind took it too far by basically re-allowing any CSS property. How doesn't that go against their own principles?